frontend/src/actionCreators/mineActionCreator.js

```diff
@@ -284,3 +284,23 @@ export const updateExpectedDocument = (id, payload) => (dispatch) => {
       dispatch(hideLoading("modal"));
     });
 };
+
+export const fetchMineBasicInfoList = (mine_guids) => (dispatch) => {
```

Comment:
out of curiosity, what is returned in the mineBasicInfoList? the naming leads me to assume it is info such as the mine tombstone data.